app/models/zone.rb

```diff
@@ -56,10 +54,8 @@ def miq_region
   end
 
   def ntp_settings
-    # Return ntp settings if populated otherwise return the defaults  {:ntp => {:server => ['blah'], :timeout => 5}}
-    return settings[:ntp] if settings.fetch_path(:ntp, :server).present?
-
-    Zone::DEFAULT_NTP_SERVERS.dup
+    # Return ntp settings if populated otherwise return the defaults
+    settings.fetch_path(:ntp, :server).present? ? settings[:ntp] : settings_for_resource.ntp.to_h
```

Comment:
How can it *not* be present if you have it in the default.

Reply:
Yeah... not confusing at all, right?